### DISCUSSION TITLE
Problem with database

### DISCUSSION THREAD

**Discussion by zvshka** [nextauthjs/next-auth#1430] [Help]
**Your question**
How to provide new values in mongodb with oauth2 provider?

**What are you trying to do**
I am using discord provider and i need id of user to search in my own database after auth, but i cant figure out how to get id of signed user
**Reproduction**

**Feedback**
*Documentation refers to searching through [online documentation](https://next-auth.js.org), code comments and issue history. The example project refers to [next-auth-example](https://github.com/iaincollins/next-auth-example).*

* [ ] Found the documentation helpful
* [x] Found documentation but was incomplete
* [x] Could not find relevant documentation
* [ ] Found the example project helpful
* [x] Did not find the example project helpful

**Comment by zvshka** [comment]
I have basic setup with database and without jwt

**Comment by zvshka** [comment]
https://hastebin.com/fewofocudo.js - main page, where i need to get id and other staff
https://hastebin.com/bopexoxijo.js - [...nextauth].js file

**Comment by StefanSelfTaught** [comment]
I would use the mongodb driver for node or mongoose.
